charts/velero/values.yaml

```diff
@@ -9,6 +9,8 @@ image:
   tag: v1.2.0
   pullPolicy: IfNotPresent
 
+imagePullSecrets: []
```

Comment:
Please add documentation to this fields so we know what's for. Thank you!

Reply:
Done